desktop/libs/indexer/src/indexer/indexers/sql.py

```diff
@@ -303,11 +303,17 @@ def create_table_from_local_file(self, source, destination, start_time=-1):
         if source['format']['hasHeader']:
           list_of_tuples = list_of_tuples[1:]
 
-        csv_rows = str(list_of_tuples)[1:-1]
-        sql += '''INSERT INTO %(table_name)s VALUES %(csv_rows)s;'''% {
-            'table_name': table_name,
-            'csv_rows': csv_rows
-          }
+        list_of_tuples = list_of_tuples[:10000]
+
+        for count in range(1 + len(list_of_tuples)//1000):
+          temp_list = list_of_tuples[count*1000:(count+1)*1000]
+          if(len(temp_list)==0):
+            break
+          csv_rows = str(temp_list)[1:-1]
+          sql += '''\n\nINSERT INTO %(table_name)s VALUES %(csv_rows)s;'''% {
+              'table_name': table_name,
+              'csv_rows': csv_rows
+            }
```

Comment:
As we discussed that we will add data till 10000 rows and they must be add in 10 batches so here is the code for that.